uaclient/config.py

```diff
@@ -288,7 +288,9 @@ def _attached_status(self) -> "Dict[str, Any]":
                 contractInfo["effectiveTo"], "%Y-%m-%dT%H:%M:%SZ"
             )
 
-        resources = get_available_resources(self)
+        resources = self.machine_token.get("availableResources", [])
```

Comment:
A total nit, but no need to instantiate a list here, the value won't be used if we go down that path. Look how much more efficient using `None` is:

```
$ python3 -m timeit '{}.get("foo", [])'
5000000 loops, best of 5: 75.1 nsec per loop
$ python3 -m timeit '{}.get("foo", None)'
5000000 loops, best of 5: 60.5 nsec per loop
```

:joy: